internal/pkg/manifest/workload.go

```diff
@@ -481,6 +481,11 @@ type ServiceConnectBoolOrArgs struct {
 	ServiceConnectArgs
 }
 
+// ServiceConnectEnabled returns if ServiceConnect is enabled or not.
+func (s *ServiceConnectBoolOrArgs) ServiceConnectEnabled() bool {
```

Comment:
```suggestion
func (s *ServiceConnectBoolOrArgs) Enabled() bool {
```
maybe just this since it's being used like `Connect.Enabled()` which seems descriptive enough?